xcp_d/workflow/execsummary.py

```diff
@@ -31,6 +31,7 @@ def init_brainsprite_wf(
     layout,
     fmri_dir,
     subject_id,
+
```

Comment:
Seems to be an added empty line. Did you want to put dummyvols in here?

Reply:
seems like a typo